### ISSUE TITLE
add option to configure service-tab order

### ISSUE THREAD

**Issue by yafp** [yafp/ttth#76]
users should be able to change the service-tab order

**Comment by yafp** [comment]
as small optimization it could help to change the function which generates the user-server-configs filename

from
> randomString_serviceType

to:
> service_Type_randomString

This should result in alphabetical sort order of the tabs

**Comment by yafp** [comment]
the small optimization will be introduced in 1.5.0
